platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -214,6 +214,20 @@ export default class HangingProtocolService extends PubSubService {
     };
   }
 
+  public filterSeriesRequiredForRun(protocolId, seriesPromises) {
```

Comment:
Please add docs here, also explaining that this can be extended in the future with more complex selection rules, eg N series of a given type, and M of a different type, such as all CT series, and all SR, and then everything else.